src/transformers/generation/__init__.py

```diff
@@ -40,6 +40,11 @@
         "BeamSearchScorer",
         "ConstrainedBeamSearchScorer",
     ]
+    _import_structure["candidate_generator"] = [
+        "AssistedCandidateGenerator",
+        "CandidateGenerator",
+        "PromptLookupCandidateGenerator",
+    ]
```

Comment:
These are now importable as `from transformers.generation import ...` (not as `from transformers import ...`). This is needed for tests (see below), but I don't think they are worth a global import.